include/faker-cxx/word.h

```diff
@@ -23,12 +27,30 @@ namespace faker::word
  * faker::word::sample(5) // "spell"
  * @endcode
  */
-FAKER_CXX_EXPORT std::string_view sample(std::optional<unsigned> length = std::nullopt);
+FAKER_CXX_EXPORT std::string_view sample(std::optional<unsigned> length = std::nullopt,const faker::Locale locale = Locale::en_US);
```

Comment:
No need for faker prefix as we are in faker namespace, also const is not required as its copy anyway

Reply:
fixed.